library/globals.inc.php

```diff
@@ -2968,6 +2968,20 @@ function gblTimeZones()
             '0',
             xl('Enable OpenEMR FHIR System Scopes.')
         ),
+        
+        'google_signin_enabled' => array(
+            xl('Enable Google Sign-In'),
+            'bool',
+            '0',
+            xl('Enable Authentication Using Google Sign-in')
+        ),
+
+        'google_signin_client_id' => array(
```

Comment:
is this considered sensitive? (ie. does it need to be stored in encrypted form?)

Reply:
disregard above question. this is being outputted on the login.php script, so assuming is not sensitive information,

Reply:
disregarded :)